src/index.html

```diff
@@ -4,9 +4,384 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="body-content">
+    <header class="header">
+      <a href="#" class="header__logo">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__Item">
+            <a class="nav__link nav__link--active is-active" href="#">Apple</a>
+          </li>
+
+          <li class="nav__Item">
+            <a class="nav__link " href="#">Samsung</a>
+          </li>
+
+          <li class="nav__Item">
+            <a class="nav__link" href="#">Смартфоны и телефоны</a>
+          </li>
+
+          <li class="nav__Item">
+            <a
+              class="nav__link "
+              data-qa="nav-hover"
+              href="#"
+            >
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+
+          <li class="nav__Item">
+            <a class="nav__link" href="#">Гаджеты</a>
+          </li>
+
+          <li class="nav__Item">
+            <a class="nav__link" href="#">Планшеты</a>
+          </li>
+
+          <li class="nav__Item">
+            <a class="nav__link" href="#">Фото</a>
+          </li>
+
+          <li class="nav__Item">
+            <a class="nav__link" href="#">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="body-content__main">
+      <section class="catalog">
+        <div class="card catalog__card" data-qa="card">
+          <img
+            class="card__img"
+            src="./images/imac.jpeg"
+            alt="Product"
+          >
+          <h2 class="card__product-name">
+            Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+          </h2>
+
+          <p class="card__product-code">Код товара: 195434</p>
+
+          <div class="card__raiting">
+            <div class="stars">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star stars__star--5"></div>
+            </div>
+
+            <p class="card__review">Отзывов: 5</p>
+          </div>
+
+          <div class="card__prise">
+            <div class="card__prise-name">Цена:</div>
+            <div class="card__prise-num">69 999 грн</div>
```

Comment:
[Checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md) 1st point. What tag do we use for text?